server/models/Notification.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: 'User'
+  },
+  type: {
+    type: String,
+    enum: ['request', 'payment', 'message'],
+    required: true
+  },
+  title: {
+    type: String,
+    default: "",
+    required: true
+  },
+  description: {
+    type: String,
+    default: "",
+        required: true
+  },
+  date: {
```

Comment:
Alternatively, you can use the timestamps option from mongoose :)

Ref: https://masteringjs.io/tutorials/mongoose/timestamps